src/components/AdminDashboard/AddIncident.js

```diff
@@ -5,8 +5,8 @@ import moment from 'moment';
 
 import {
```

Comment:
Great work, Kelly, we saw the display, Let's try to push this one and reconcile the codebase with the DS codebase (also in need of much reconciliation).